test_pii_detect.py

```diff
@@ -44,6 +44,12 @@ def test_aggie_proud(self):
         # merge errors are a common reason for failures
         result_list = show_aggie_pride()
         self.assertEqual(result_list[3], 'We are Aggies!')
+
+    def thats_on_1891(self):
```

Comment:
Sorry I missed this last night - test functions need to begin with the word test or they won't run in the test harness

test_thats_on_1891() would work